languages/en-US/moderation/unban.json

```
@@ -3,6 +3,6 @@
     "USAGE": "{{prefix}}unban [userID/user#0000]",
     "EXAMPLES": "{{prefix}unban 422820341791064085\n{{prefix}}unban Androz#2091",
     "MISSING_ID": "Please specify the ID of the member you wish to unban!",
-    "NOT_BANNED": "**{{username}}** is not banned!",
+    "NOT_BANNED": "**{{username}}** is not banned! (yet)",
```

Si la personne veut l'unban, c'est qu'il n'est pas prêt d'être banni non ?
```suggestion
    "NOT_BANNED": "**{{username}}** is not banned!",
```

---

Pas faux.. Je voulais faire une vanne mais tu me casse comme ça, rudement.